module/VuFind/src/VuFind/Form/Form.php

```diff
@@ -1089,4 +1103,40 @@ protected function getElementId(string $id): string
     {
         return 'form_' . $this->formConfig['id'] . '_' . $id;
     }
+
+    /**
+     * Get primary form handler
+     *
+     * @return HandlerInterface
+     */
+    public function getPrimaryHandler(): HandlerInterface
+    {
+        $handlerName = ($this->formConfig['primaryHandler'] ?? 'email');
+        return $this->handlerManager->get($handlerName);
+    }
+
+    /**
+     * Get secondary form handlers
+     *
+     * @return HandlerInterface[]
+     */
+    public function getSecondaryHandlers(): array
+    {
+        $handlerNames = (array)($this->formConfig['secondaryHandlers'] ?? []);
+        $handlers = [];
+        foreach ($handlerNames as $handlerName) {
+            $handlers[] = $this->handlerManager->get($handlerName);
+        }
+        return $handlers;
+    }
+
+    /**
+     * Get current form id/name
+     *
+     * @return string
+     */
+    public function getFormId(): string
```

Comment:
Would it make sense to use this internally for other references to `$this->formConfig['id']` (e.g. in `getElementId` above)?